example/README.md

```diff
@@ -0,0 +1,48 @@
+# LNC Example HTML Page
+
+This folder contains an example HTML page with JS code used to demonstrate how
+to connect to a `litd` node using LNC on the web.
+
+## Setup
+
+Follow these steps to be able to view the example page in a browser on your
+machine.
+
+1. Clone this repository
```

Comment:
nit: seems like the numbered lists here did not auto-increment 🤓

Reply:
The increments are working on Github for me. Where are you seeing the increments not working? I can manually change it but just curious.

https://github.com/jamaljsr/lightning-node-connect/tree/example-readme/example

![image](https://user-images.githubusercontent.com/1356600/197018326-02a9aaf5-68d1-4747-ae7b-af4b67c1d101.png)